### ISSUE TITLE
"epoch" column not present in my SQLite database

### ISSUE THREAD

**Issue by siriusb-nox** [faircloth-lab/tapir#10]
Dear TAPIR development team,

After successfully running tapir on UNIX and have plotted the pi profiles for all loci, I am trying now to plot loci PI per time interval in R (the python script won't run because I have an obsolete rpy2 version). However, when I execute:
> pi <- dbGetQuery(con, "select loci.locus, epoch, sum_integral from loci, epoch where 
+     loci.id = epoch.id")

I get the following error message:
Error in result_create(conn@ptr, statement) : no such table: epoch

When I look at the tables nested in my sql file, I get:
> dbListTables(con)
[1] "discrete"        "interval"        "loci"            "net"            
[5] "sqlite_sequence"

So in short, it seems "epoch" is not being included on the SQLite database during analysis. Do you know how to sort this out?

Many thanks again for any help you could provide!

Best,

Oscar

**Comment by siriusb-nox** [comment]
I found an alternative way to get the intervals plot! However, the column "epoch" is still missing in the SQL file produced by tapir.

Thanks,

Oscar
